src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -225,17 +225,17 @@
   <data name="Generate_all" xml:space="preserve">
     <value>Generate all</value>
   </data>
-  <data name="Generate_enum_member_1_0" xml:space="preserve">
-    <value>Generate enum member '{1}.{0}'</value>
+  <data name="Generate_enum_member_0" xml:space="preserve">
+    <value>Generate enum member '{0}'</value>
   </data>
   <data name="Generate_constant_1_0" xml:space="preserve">
     <value>Generate constant '{1}.{0}'</value>
   </data>
   <data name="Generate_read_only_property_1_0" xml:space="preserve">
     <value>Generate read-only property '{1}.{0}'</value>
   </data>
-  <data name="Generate_property_1_0" xml:space="preserve">
```

Comment:
xlf files need to be generated

Reply:
yeah, my VS is having issues freezing right now so I needed to restart